module/Motion/CorrectingRotation.cpp

```diff
@@ -56,6 +57,7 @@ void CorrectingRotation::run()
    * 目標角度を計測角度の方向に90度ずらして再設定する
    */
   if(abs(tAngle - measuredAngle) > 45) {
+    printf("横線を採用！！！\n");
```

Comment:
実験用なら消して、残すつもりならlogger使っておくれ。
```suggestion
    logger.log("横線を採用！！！\n");
```